nvdaHelper/vbufBackends/adobeAcrobat/adobeAcrobat.cpp

```diff
@@ -56,16 +55,20 @@ long getAccID(IServiceProvider* servprov) {
 	} 
 	LOG_DEBUG(L"IAccID at "<<paccID);
 
+	// IAccID::get_accID takes a longlong on 64 bit and a long on 32 bit.
+	// However, Acrobat will internally only place a 32 bit value into ID.
+	// Thus we call it with a LONG_PTR* and can safely static cast it to a long.
+	LONG_PTR ID = 0;
 	LOG_DEBUG(L"Calling get_accID");
-	if((res=paccID->get_accID((long*)(&ID)))!=S_OK) {
+	if((res=paccID->get_accID(&ID))!=S_OK) {

```

Comment:
Minor linting fix for readability

```suggestion
	if ((res = paccID->get_accID(&ID)) != S_OK) {
```